tests/decay/test_descriptor.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+import pytest
+
+from decaylanguage import DecayChain, DecayMode
+from decaylanguage.utils import DescriptorFormat
```

Comment:
Since this test is for something under utils, move the test file under tests/utils, for consistency with the rest.